apps/remote_control/lib/lexical/remote_control/dispatch/handlers/indexing.ex

```diff
@@ -8,9 +8,20 @@ defmodule Lexical.RemoteControl.Dispatch.Handlers.Indexing do
   require Logger
   import Messages
 
-  use Dispatch.Handler, [file_compile_requested(), filesystem_event()]
+  use Dispatch.Handler, [filesystem_event(), file_changed(), file_opened()]
 
-  def on_event(file_compile_requested(uri: uri), state) do
+  def on_event(file_changed(uri: uri, open?: true), state) do
```

Comment:
why do we care if the file is open to index it?

Reply:
Okay, we can reindex all the renamed files once the renaming is complete